### DISCUSSION TITLE
field.valueAsString returning dropdown value, not label

### DISCUSSION THREAD

**Discussion by Coysh** [solspace/craft-freeform#1090] [Questions & Answers]
**Describe the bug or issue you're experiencing**
Our form stores GUIDS in a dropdown as the value, so we want the template to return the label value.

**Steps to reproduce**

1. Create dropdown
2. Create template
3. Use {{ field.valueAsString }}

Code snippet:
`<strong>{{ field.label }}:</strong> {% if (field.handle == 'primaryTitle') or (field.handle == 'leaderTitle') or (field.handle == 'referee1Title') or (field.handle == 'referee2Title') %}{{ field.valueAsString }}{% else %}{{ field.getValueAsString() }}{% endif %}`

**Expected behavior**
Output should be "Mr", where it is GUID

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.6.0 Pro
- Freeform Version: 5.0.10
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Additional context**
Unsure if this was working in Freeform 4

**Comment by kjmartens** [comment]
Hi @Coysh,

Would you be able to provide a bit more context for this? I am not 100% sure what you're doing.

Also, as a side note, Freeform 5 introduces `field.labels` and `field.labelsAsString`, which allows you to access option labels vs option values. If your conditionals are relying on an option label instead of an option value, it's possible this could be at play here. 🙂 
https://docs.solspace.com/craft/freeform/v5/setup/updating-freeform-4/
